<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello Game - Start a game</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../.designer/backup/res/favicon.jpg</normaloff>../../../../../.designer/backup/res/favicon.jpg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="gamename">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ff0000;&quot;&gt;H&lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffaa00;&quot;&gt;l&lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;l&lt;/span&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#55aa00;&quot;&gt;o&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;/&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="serverValue">
     <property name="placeholderText">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nicknameValue">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nickname</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="errorMsgText">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ConnectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
